MapboxNavigation/Constants.swift

```diff
@@ -1,6 +1,35 @@
 import Foundation
+import MapboxDirections
+
+typealias CongestionSegment = ([CLLocationCoordinate2D], CongestionLevel)
+
+/**
+ Line width base values used at zoom levels.
+ */
+public let lineWidthAtZoomLevels: [Int: MGLStyleValue<NSNumber>] = [
+    10: MGLStyleValue(rawValue: 8),
+    13: MGLStyleValue(rawValue: 9),
+    16: MGLStyleValue(rawValue: 11),
+    19: MGLStyleValue(rawValue: 22),
+    22: MGLStyleValue(rawValue: 28)
+]
+
+/**
+ Default opttions applied to features in NavigationMapView.
+ */
+public let sourceOptions: [MGLShapeSourceOption: Any] = [.maximumZoomLevel: 16]
```

Comment:
How is the developer expected to use this constant? Are they expected to use these options for all shape sources they create?